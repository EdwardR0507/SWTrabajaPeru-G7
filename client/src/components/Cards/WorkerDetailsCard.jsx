import { Box, Card, CardHeader, Avatar, ListItem, ListItemAvatar, ListItemText } from "@material-ui/core";
import CardContent from "@material-ui/core/CardContent";
import PhoneAndroidIcon from "@material-ui/icons/PhoneAndroid";
import MailIcon from "@material-ui/icons/Mail";
import RoomIcon from "@material-ui/icons/Room";

export default function ProfileCard() {
  return (
      <Card>
        <Box
          style={{
            textAlign: "center",
            display: "flex",
            justifyContent: "center",
            alignItems: "center",
          }}
        >
        <CardHeader
            avatar={
              <Avatar
                style={{ width: 100, height: 100, borderRadius: 75 }}   
              ></Avatar>
            }
          />
        </Box>

        <CardHeader
          title="Kori Antunez Palomino"
          style={{ textAlign: "center" }}
        />
        <CardContent>
          <ListItem>
            <ListItemAvatar>
              <Avatar>
                <PhoneAndroidIcon />
              </Avatar>
            </ListItemAvatar>
            <ListItemText primary="Teléfono / Celular" secondary="923585728" />
          </ListItem>
          <ListItem>
            <ListItemAvatar>
              <Avatar>
                <MailIcon />
              </Avatar>
            </ListItemAvatar>
            <ListItemText
              primary="Correo electrónico"
              secondary="kori.antunez@gmail.com"
            />
          </ListItem>
          <ListItem>
            <ListItemAvatar>
              <Avatar>
                <RoomIcon />
              </Avatar>
            </ListItemAvatar>
            <ListItemText
              primary="Ubicación"
              secondary="Lima/Lima/Jesús María"
            />
          </ListItem>          
        </CardContent>
      </Card>
  );
}
