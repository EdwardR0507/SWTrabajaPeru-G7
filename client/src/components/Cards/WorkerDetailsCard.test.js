/*Importamos las librerias principales*/
import React from "react";
import { shallow } from "enzyme";
import WorkerDetailsCard from "./WorkerDetailsCard";
/*Renderizado del componente WorkerDetailsCard*/
describe("<WorkerDetailsCard></WorkerDetailsCard>", () => {
    it('render "WorkerDetailsCard"', () => {
        const wrapper = shallow(<WorkerDetailsCard />);
        expect(wrapper).toMatchSnapshot()
      });
});
