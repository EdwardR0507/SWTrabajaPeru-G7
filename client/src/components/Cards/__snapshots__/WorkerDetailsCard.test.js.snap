// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WorkerDetailsCard></WorkerDetailsCard> render "WorkerDetailsCard" 1`] = `ShallowWrapper {}`;
